### ISSUE TITLE
(@aws-cdk/aws-logs): MetricFilter - No option to set property filterName

### ISSUE THREAD

**Issue by sandip-oneadvanced** [aws/aws-cdk#17626]
### General Issue

MetricFilter lacks support for filterName

### The Question

It seems that, currently CDK/Cloudframation(`AWS::Logs::MetricFilter`) does not have option to set `filterName` property when you are creating MetricFilters. 

It is available only with SDK, CLI, Rest API. These behavior leaves `filterName` being generated randomly. Are there any alternatives to provide `filterName` property while using CDK?

### CDK CLI Version

1.133.0

### Node.js Version

14.x

### OS

windows

### Language

Typescript

### Language Version

4.5.2

**Comment by rix0rrr** [comment]
You can always resort to escape hatches if CDK does not emit the CloudFormation template you are expecting: https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html

**Comment by rix0rrr** [comment]
This property should just be added

**Comment by sandip-oneadvanced** [comment]
@rix0rrr, 

the native CloudFormation resource does not support `filterName` either. Please have a look at `AWS::Logs::MetricFilter` [Cloud Formation Resource Definition](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-metricfilter.html)

I've tried to use methods mentioned in escape hatches. But, due the fact that CloudFormation doesn't support that property `cdk deploy` fails with error - `Encountered unsupported property FilterName`

**Comment by JeromeDuboisVizzia** [comment]
I agree, this feature would be quite useful to help navigate in our Metric Filters.
Any updates since 2021?
